src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs

```diff
@@ -0,0 +1,124 @@
+						public static ResourceValue<Brush> Unselected => new("RatingControlForegroundUnselected", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForegroundUnselectedPointerOver")]
+						public static ResourceValue<Brush> UnselectedPointerOver => new("RatingControlForegroundUnselectedPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForegroundSelected")]
+						public static ResourceValue<Brush> Selected => new("RatingControlForegroundSelected", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForegroundSelectedPointerOver")]
+						public static ResourceValue<Brush> SelectedPointerOver => new("RatingControlForegroundSelectedPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForegroundSelectedDisabled")]
+						public static ResourceValue<Brush> SelectedDisabled => new("RatingControlForegroundSelectedDisabled", true);
+					}
+
+					public static class PlaceholderForeground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlPlaceholderForeground")]
+						public static ResourceValue<Brush> Default => new("RatingControlPlaceholderForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlPlaceholderForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("RatingControlPlaceholderForegroundPointerOver", true);
+					}
+
+					public static class CaptionForeground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlCaptionForeground")]
+						public static ResourceValue<Brush> Default => new("RatingControlCaptionForeground", true);
+					}
+
+					public static class Typography
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "RatingControlFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("RatingControlFontFamily", true);
+
+						[ResourceKeyDefinition(typeof(FontFamily), "RatingControlCaptionFontFamily")]
+						public static ResourceValue<FontFamily> CaptionFontFamily => new("RatingControlCaptionFontFamily", true);
+
+						[ResourceKeyDefinition(typeof(Style), "RatingControlCaptionStyle")]
+						public static ResourceValue<Style> CaptionStyle => new("RatingControlCaptionStyle", true);
+					}
+
+					[ResourceKeyDefinition(typeof(double), "RatingControlHeight")]
+					public static ResourceValue<double> Height => new("RatingControlHeight", true);
+
+					[ResourceKeyDefinition(typeof(double), "RatingControlCaptionHeight")]
+					public static ResourceValue<double> CaptionHeight => new("RatingControlCaptionHeight", true);
+				}
+
+				public static class Secundary
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SecundaryRatingControlForeground")]
+						public static ResourceValue<Brush> Default => new("RatingControlForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "SecundaryRatingControlForegroundUnselected")]
```

Comment:
```suggestion
						[ResourceKeyDefinition(typeof(Brush), "SecondaryRatingControlForegroundUnselected")]
```

And the same for most of `Secondary`